### PR TITLE
GG3: Add steam DLLs and client connection

### PR DESCRIPTION
**Pull request by owencummings**
- If you have steam open, you should see your steam name in the print output & play spacewars on Steam

### CODE REVIEW

File: Scripts/Steam/SteamManager.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+using Godot;
+using System;
+using Steamworks;
+
+public partial class SteamManager : Node
+{
```
Comment (mfpardotrain) [inline Scripts/Steam/SteamManager.cs:6]: i will start using this syntax but it gives me a headache

File: Scripts/Steam/SteamManager.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	// Called when the node enters the scene tree for the first time.
+	public override void _Ready(){
+		if (Instance == null){
+			Instance = this;
+			try
+			{
+				SteamClient.Init(GameAppId, true);
+				if (!SteamClient.IsValid){
+					GD.Print("Steam client not initialized properly.");
+					throw new Exception();
```
Comment (mfpardotrain) [inline Scripts/Steam/SteamManager.cs:22]: da hell is this exception i dont know whats going on

**Review by mfpardotrain — APPROVED** [review]
wahtever

File: Scripts/Steam/SteamManager.cs (hunk 2 above)
Comment (owencummings) [reply Scripts/Steam/SteamManager.cs:22]: Yeah probably not necessary to do a whole try-except block

**Pull request merged**